airflow_clickhouse_plugin/sensors/clickhouse_sql_sensor.py

```diff
@@ -0,0 +1,23 @@
+from airflow.sensors.sql_sensor import SqlSensor
+from airflow_clickhouse_plugin.hooks import clickhouse_hook
+
+
+class ClickHouseSqlSensor(SqlSensor):
+    def __init__(
+        self,
+        clickhouse_conn_id: str = "clickhouse_default",
+        database=None,
+        *args,
+        **kwargs
+    ):
+        super(ClickHouseSqlSensor, self).__init__(conn_id=None, *args, **kwargs)
+        self._conn_id = clickhouse_conn_id
```

Comment:
Use just an initializer call:

```suggestion
        super().__init__(conn_id=clickhouse_conn_id, *args, **kwargs)
```

Without `self._conn_id = clickhouse_conn_id`. Please rely on inheritance.

Reply:
I just look at how execute method of ClickHouseOperator was implemented

Reply:
`ClickHouseOperator` inherits from `BaseOperator` which does not have `conn_id` field. `SqlSensor` has.